omniNotes/src/androidTest/java/it/feio/android/omninotes/helpers/Methods.java

```diff
@@ -0,0 +1,51 @@
+package it.feio.android.omninotes.helpers;
+
+import android.support.test.espresso.AppNotIdleException;
+import android.support.test.espresso.NoMatchingRootException;
+import android.support.test.espresso.NoMatchingViewException;
+import android.view.View;
+
+import junit.framework.AssertionFailedError;
+
+import org.hamcrest.Matcher;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.action.ViewActions.swipeLeft;
+import static android.support.test.espresso.action.ViewActions.typeText;
+import static android.support.test.espresso.assertion.PositionAssertions.isBelow;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.isCompletelyDisplayed;
+
+public class Methods {
+
+    public static void clickUIElement(Matcher<View> matcher) {
+        onView(matcher).perform(click());
+    }
+
+    public static void typeItemText(Matcher<View> matcher, String textToBeTyped) {
+        onView(matcher).perform(typeText(textToBeTyped));
+    }
+
+    public static boolean isUIObjectIsDisplayed(Matcher<View> matcher) {
```

Comment:
isUIObjectDisplayed would be a better name

Reply:
Done.